googleapis_auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.4.1-dev
+
+- Handle the `source_credentials` from the credentials file to support 
```

Comment:
Please put this under a (new) `auth_io.dart` section below (under 1.4.0-dev)

Reply:
Done